### DISCUSSION TITLE
Joining INVARCH and TNKR staking as DCDAO

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#91] [Proposals Temperature Check]
Hi community!

DCDAO needs to start thinking towards becoming a self funded DAO. As a first step towards this goal I propose we join VARCH and TNKR staking as DCDAO. Varch is token from INVARCH NETWORK which is on Polkadot relay chain and Tnkr is token from TINKER NETWORK which is on Kusama relay chain. Both are parachains like are Darwinia Network and Crab Network.

These are requirements directly copied from their Discord Channel to join:

_This channel is dedicated to new Decentralized Autonomous Organizations (DAOs) looking to apply for DAO staking registration on the InvArch network. Currently, we are adding DAOs manually, but in the future, this process will be automated and permissionless through our Dashboard UI._

To apply, please provide the following information:
1. **Name**: The official name of your DAO.
2. **Website/Social Media Profiles**: Links to your website and active social media accounts.
3. **Description**: Include details such as the history of your DAO, its purpose, the number of members, and the governance system you employ.
4. **Short Description**: Include a catchy 2-3 sentence description that we will add to your profile on our Dashboard.
5. **Operating Network**: Specify the network your DAO is currently operating on, if applicable.
6. **Logo**: Attach your DAO's logo.

_It's important to note that DAO staking aims to support DAOs that plan to operate on the InvArch network in the future and utilize our DAO tooling infrastructure. We reserve the right to reject applications that do not meet the criteria for being recognized as DAOs._

I have been following Invarch for a year and a half now mostly as a member of Youdle DAO which will be powered by Invarch tech. I have come to know the members of Invarch team to be responsive and always ready to provide help and explanation. They kept on building through bear market like did we. I think Darwinia tech can be implimented by Invarch/Youdle DAO as we can impliment their DAO tooling structure.

Here are some links regarding Invarch tooling:
https://invarch.network/
Here are the links regarding Darwinia tooling:
https://darwinia.network/

To learn more then just what is written on this link I will invite Invarch team to join our debate and answer any questions from DCDAO community. (EDITED: FORGOT THEY CAN NOT JOIN HERE TO DEBATE SO I WILL POST ANY QUESTIONS IN THEIR DISCORD AND REPLIES HERE)

I encourage all to join debate and ask questions.

Here are two of mine that I came across when doing research:

- Cap to start earning rewards with VARCH is 250000 Varch and with TNKR 25000 Tnkr. I think to attract users we will need to share some rewards with them(similar to what some projects in Astar dApp staking do). We could have a joined AMA together on launch(Jesse proposed this good idea that since he has a DM channel open with Dakota founder of Invarch and he will try to meet with him in ETH Denver) to introduce both communities and introduce what we as DCDAO do and what INVARCH does. I see nice collaboration with Darwinia and Invarch in future, possible YoudleDAO and DCDAO too. But let's not rush;-)
- We hand pick our members so hopefully much smaller DCDAO members number than other DAOs might not cause an issue.

This debate will be on for one week with option to extend for one more week if needed or requested.

**Comment by 2xJDubs** [comment]
I think we do need to understand what is required by operating on the InvArch Network and Utilize their DAO tooling as this may not work for us but it could if we know more.  I like the idea of using the InvArch DAO staking to help bolsters DCDAO's funding so it is not as reliant on Darwinia for funding.  I am excited with what InvArch is building and would love to see more collaboration as well.

**Comment by SasoLithops** [comment]
Very valid points @2xJDubs . 
I asked in Dao application channel in their Discord and got answer no cost necessary for using Saturn multisig account at this time and no other requirements. I attach discussion.

![Screenshot_2024-02-17-12-44-05-927_com.discord.jpg](https://github.com/dcdao/discussion/assets/101176902/dc561c27-9ca3-42fc-9db5-f35d6b4f24f8)

![Screenshot_2024-02-17-12-44-38-969_com.discord.jpg](https://github.com/dcdao/discussion/assets/101176902/784b3516-df2c-4368-ad9e-1f5e1eba8d1b)

**Comment by hackfisher** [comment]
i'm good with this idea, as Darwinia Msgport is also looking for adoption of cross-chain governance use cases, probably we can looking forward something more exciting by interact with INVARCH, opportunities to adopt Darwinia Msgport in their uses cases for multichain govenance to more broad EVM chains.

https://docs.darwinia.network/multi-chain-dao-governance-d3b2e194828d4af8bad8e3fa28219fc3 cc @wuminzhe 

Requesting more informations about what need to be supported or which resources are needed, and what it can be used for (Managing assets in other parachains for DCDAO)?

**Reply by SasoLithops** [reply]
Awesome. I will forward this in their DAO adoption thread to get answer. I can connect you with them if you wish to learn more.

**Comment by SasoLithops** [comment]
@hackfisher 

This is the answer I got. Let me know if you have more.

![Screenshot_2024-02-19-16-36-12-244_com.discord.jpg](https://github.com/dcdao/discussion/assets/101176902/c9586365-3d7e-4ea2-835e-9ef73965cdf4)

**Reply by 2xJDubs** [reply]
I think in a way Msgport and the Saturn multi-sig are competing product but I do wonder if some of the benefits in Msgport can help the Saturn multi-sig.

**Comment by hackfisher** [comment]
Firstly, my understanding is that Darwinia xAccount is doing something to EVM chains similar to what INVARCH does for Polkadot chains, which is account abstract(AA) in the cross-chain level, aka. one account for any chain, for EVM or Polkadot blockchains. Currently, we are targeting different markets: Darwinia xAccount is focused on EVM chains, while INVARCH targets Substrate chains. This is why I can grasp their vision immediately and understand how they work. I believe we are not competitors yet and could collaborate to expand into a broader market by integrating each other's services. For example:

1. Darwinia xAccount users could interact with Polkadot chains by integrating Darwinia xAccount/chain with INVARCH.
2. INVARCH Saturn Multisig users could interact with EVM chains using their Saturn Multisig account if Darwinia xAccount and INVARCH are integrated through Darwinia chain. (I still have no idea INVARCH is supporting runtime only or also support ink! smart contract?)

Since these two account systems are heterogeneous, integrating them might present challenges and require investment, which may result in it being a lower priority in the short term as we are still making inroads into each other's markets. However, we can share similar solutions and possibly cooperate to simplify the conceptualization process.

Secondly, regarding your proposal, it appears to be limited to the scope of DCDAO, as there are no resource barriers, and it only requires creating a multisig account on INVARCH and submitting some proposals. I'm comfortable with that, and you may discuss content and plans with the other Community Managers, or achieve consensus through voting or proposals, and then proceed to act on behalf of DCDAO.

And, may I know how DAO Staking is working, does it means INVARCH are allocating tokens to these DAOs by some form of staking? What's it thesis for design both for DAO and INVARCH token? Or could you provide more documents about DAO Staking?

**Reply by SasoLithops** [reply]
Sorry for late reply, I guess you edited the questions later.

Cap to start earning rewards as a DAO with VARCH is 250000 Varch and with TNKR 25000 Tnkr. They have certain percentage of tokens for rewarding those DAOs and people who stake these tokens to these DAOs. 
I think to attract stakers we will need to share some rewards with them(similar to what some projects in Astar dApp staking do). I think that because it will be for most VARCH,TNKR token holders imho first time they hear of DCDAO and I find it impossible they will freely nominate tokens to DCDAO. 
Another idea maybe we could also give commemorative NFT which could monthly give some small % of TNKR/VARCH and also Pink(we could apply for 5%), maybe also some Ring, Crab incentives to attract users to our eco as well.  
Any other idea more then welcome. 

Let me ask Mikhail from their team to provide more detailed answers on your questions will post here when he does.

**Reply by SasoLithops** [reply]
DAO Staking, Farming, & Funding — Rather than stake towards a validator to secure the network, since shared-security is provided by the Polkadot relay, VARCH holders are able to stake towards registered DAOs on the network in order to support their growth.

In the future, users will be able to stake toward DAOs on the network for staking rewards, but then also redirect their staking rewards to other DAOs on the network in exchange for rewards (such as fungible & non-fungible tokens) offered by those DAOs.

The VARCH supply experiences a 10% annual inflation rate, of which 60% goes towards funding DAOs & 40% goes towards rewarding stakers. This system is designed to realize a flywheel effect where users of the networks are able to fund other users to contribute services to the network, and make the network more valuable for everyone as a result. This also gives the power of directing monetary policy to network stakers.

From medium article:

https://invarch.medium.com/invarch-varch-tde-details-df69c9db7783

**Reply by SasoLithops** [reply]
His reply:

Holders of both VARCH and TNKR tokens will have the opportunity to stake their tokens towards your DAO, earning rewards in the process. Of these rewards, 60% will be directed towards funding your DAO, while the remaining 40% will reward the stakers themselves. These funds can be claimed, re-staked, or withdrawn by your DAO through our multisig feature.

As for the technical details regarding account abstraction, our immediate plan involves deploying our Saturn Gateway on Kusama, followed by an implementation on Polkadot. Subsequently, after developing our TSS protocol, you'll gain the capability to manage funds across any chain via the Saturn Dashboard. While I can't provide specific details on the 'how' and 'when' just yet, please stay tuned for future updates!

**Comment by SasoLithops** [comment]
I will consult with fellow ambassadors(and anyone willing to join in debate here) on how the proposal should be finalised and submit proposal for voting next week.
